extensions/resteasy-classic/rest-client-config/runtime/src/main/java/io/quarkus/restclient/config/RestClientsConfig.java

```diff
@@ -1,8 +1,8 @@
 package io.quarkus.restclient.config;
 
-import java.util.HashMap;
 import java.util.Map;
 import java.util.Optional;
+import java.util.concurrent.ConcurrentHashMap;
 
 import org.wildfly.common.annotation.NotNull;
```

Comment:
Can you please remove this while you're at it. Thanks!

Reply:
Will do shortly

Reply:
Done